README.md

```diff
@@ -157,7 +157,7 @@ func main() {
 		}
 		return
 	}
-	fmt.Println(resp.Content[0].Text)
+	fmt.Println(*resp.Content[0].Text)
```

Comment:
Prefer a `.GetText()` to a direct dereference.

Reply:
Fixed